packages/upload-api/src/blob/add.js

```diff
@@ -5,7 +5,6 @@ import * as Blob from '@web3-storage/capabilities/blob'
 import * as W3sBlob from '@web3-storage/capabilities/web3.storage/blob'
 import * as HTTP from '@web3-storage/capabilities/http'
 import * as API from '../types.js'
-
```

Comment:
could we take this out to avoid a `upload-api` release?

Reply:
addressed